src/renderer/containers/AddMetadataPage/index.tsx

```diff
@@ -0,0 +1,137 @@
+import { Alert, Spin } from "antd";
```

Comment:
The majority of the code in this file is from ye olde [`<UploadWithTemplatePage />`](https://github.com/aics-int/aics-file-upload-app/blob/main/src/renderer/containers/UploadWithTemplatePage/index.tsx)